<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>菜单1</string>
    </property>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>菜单2</string>
    </property>
    <addaction name="action_21"/>
    <addaction name="action_22"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>菜单3</string>
    </property>
    <addaction name="action_31"/>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_11">
   <property name="text">
    <string>菜单项11</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>菜单项21</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>菜单项22</string>
   </property>
  </action>
  <action name="action_31">
   <property name="text">
    <string>菜单项31</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
